src/standard_models.jl

```diff
@@ -72,3 +72,16 @@ function model_SCAN(lattice::AbstractMatrix, atoms::Vector{<:Element},
                     positions::Vector{<:AbstractVector}; kwargs...)
     model_DFT(lattice, atoms, positions, [:mgga_x_scan, :mgga_c_scan]; kwargs...)
 end
+
+"""
+Build an PBE0 model from the specified atoms.
+DOI:10.1103/PhysRevLett.77.3865
+https://www.vasp.at/tutorials/latest/hybrids/part1/
+"""
+function model_PBE0(lattice::AbstractMatrix, atoms::Vector{<:Element},
```

Comment:
Also needs to be added in https://github.com/JuliaMolSim/DFTK.jl/blob/master/src/external/atomsbase.jl

Reply:
Maybe also add a Hartree-Fock model?